docs/getting-started-guides/locally.md

```diff
@@ -17,55 +24,138 @@ At least [Docker](https://docs.docker.com/installation/#installation)
 ps`).  Some of the Kubernetes components need to run as root, which normally
 works fine with docker.
 
-#### etcd
-
-You need an [etcd](https://github.com/coreos/etcd/releases) in your path, please make sure it is installed and in your ``$PATH``.
-
-#### go
-
-You need [go](https://golang.org/doc/install) at least 1.3+ in your path, please make sure it is installed and in your ``$PATH``.
-
 ### Starting the cluster
```

Comment:
Please, pack the "Starting the cluster" section into `hack/local-up-cluster.sh` file in the `kubernetes` repo (submit a parallel PR).